Brick-Studios/BeastArena
# Animation

# Description

You can now add animation to textures.

How to add animation:
Make a texture with 2 sprites on it
So instead of this:
![elephant-1](https://user-images.githubusercontent.com/33418574/70522787-173ebb00-1b42-11ea-81b9-56584e93f417.png)

Do this:
![elephant-idle](https://user-images.githubusercontent.com/33418574/70522775-1148da00-1b42-11ea-93ab-b0991dea364c.png)

Then you call the other createImage with a sprite_width + height (how thick the elephant is), the update_time (how often you want it to animate) and sprite_size (how many elephants are in the texture)

# How can this be tested?
Test with https://github.com/Brick-Studios/BrickEngine/pull/63
Launch the game, select a player and see if they are animated, also test this by going through the levels and see if they keep the animation correctly.

# Checklist:
